src/stores/__tests__/bots.spec.js

```diff
@@ -0,0 +1,83 @@
+import { setActivePinia, createPinia } from 'pinia'
+
+import {
+	getConversationBots,
+	disableBotForConversation,
+	enableBotForConversation,
+} from '../../services/botsService.js'
+import { generateOCSResponse } from '../../test-helpers.js'
+import { useBotsStore } from '../bots.js'
+
+jest.mock('../../services/botsService', () => ({
+	getConversationBots: jest.fn(),
+	disableBotForConversation: jest.fn(),
+	enableBotForConversation: jest.fn(),
+}))
+
+describe('botsStore', () => {
+	beforeEach(async () => {
+		setActivePinia(createPinia())
+	})
+
+	it('returns an empty array if bots are not available', async () => {
+		const botsStore = useBotsStore()
+		expect(botsStore.getConversationBots('empty-token')).toEqual([])
+	})
```

Comment:
It says that it tests stores when "bots are not available" but neither this test nor others actually touch bots "availability". The store doesn't check the capability `bots-v1`.

I think it should be renamed to something like `returns empty bots array when they not loaded`. Then it would also document not obvious behavior - that both conversation without bots and conversation with not loaded bots have `[]` as bots list, bot `[]` and `null`.